### PR TITLE
CFI report change to allow for better selection of who

### PR DESCRIPTION
**Pull request by tim-hogan**
One file change only, simple change to CFI report.

### CODE REVIEW

File: MedicalBfr.php [hunk 1]
```diff
@@ -65,8 +65,7 @@ function printit(){window.print();}
 <?php
 $DEBUG=0;
 $diagtext="";
-$con_params = require('./config/database.php'); $con_params = $con_params['gliding']; 
-$con=mysqli_connect($con_params['hostname'],$con_params['username'],$con_params['password'],$con_params['dbname']);
+$con=mysqli_connect("127.0.0.1","admin","Checkers305","gliding");
```
Comment (dan-corneanu) [inline MedicalBfr.php]: @tim-hogan what is wrong with using ```$con_params```? is it failing to connect under certain circumstances?
I really do not think it is a good idea to have the  database admin password published in GIT.

**Comment by tim-hogan** [comment]
Agree, shouldn’t  have been in there.

Not sure why or how it got there.

Needs to be removed to use params.

Tim

**Comment by dan-corneanu** [comment]
@tim-hogan I have removed database password from the file an reverted back to using database connection parameters.
I have reviewed the rest of the changes and I am happy to merge and deploy them this evening.

However, there are two uncommitted changes in production

- MedicalBfr2.php
- TestMess.php

these will be deleted when deploying the latest release branch. Do you need these files?

**Comment by tim-hogan** [comment]
TestMess.php Is a quick test for Brian, will be replaced by a branch and pull request in weekend.
MedicalBfr2.php was another test fro Brian and Mark, its now been replaced so can go.

Cheers

**Pull request merged**